velox/benchmarks/basic/CastBenchmark.cpp

```diff
@@ -32,7 +32,17 @@ int main(int argc, char** argv) {
   auto vectorMaker = benchmarkBuilder.vectorMaker();
   auto emptyInput = vectorMaker.flatVector<facebook::velox::StringView>({""});
   auto validInput = vectorMaker.flatVector<facebook::velox::StringView>({""});
-  auto nanInput = vectorMaker.flatVector<facebook::velox::StringView>({""});
+  auto invalidInput = vectorMaker.flatVector<facebook::velox::StringView>({""});
```

Comment:
I'm not sure why this benchmark is written that way, but this pattern is not ideal. Let's use

```
vectorMaker.flatVector<std::string>(vectorSize, [] (auto row) {return ...;});
```